homeassistant/components/overseerr/sensor.py

```diff
@@ -0,0 +1,106 @@
+"""Support for Overseerr sensors."""
+
+from collections.abc import Callable
+from dataclasses import dataclass
+
+from python_overseerr import RequestCount
+
+from homeassistant.components.sensor import (
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import REQUESTS
+from .coordinator import OverseerrConfigEntry, OverseerrCoordinator
+from .entity import OverseerrEntity
+
+
+@dataclass(frozen=True, kw_only=True)
+class OverseerrSensorEntityDescription(SensorEntityDescription):
+    """Describes Overseerr config sensor entity."""
+
+    value_fn: Callable[[RequestCount], int]
+
+
+SENSORS: tuple[OverseerrSensorEntityDescription, ...] = (
+    OverseerrSensorEntityDescription(
+        key="total_requests",
+        native_unit_of_measurement=REQUESTS,
+        state_class=SensorStateClass.TOTAL,
```

Comment:
I'm wondering if some of those might actually be `TOTAL_INCREASING`

Reply:
I think you can delete them, so they don't have to be strictly increasing

Reply:
is this the total requests (as in, ever made), or currently active?

Reply:
ever made, but they can be deleted and that erases it

Reply:
But wouldn’t that mean a new „meter“ cycle and therefore still be total_increasing?

Reply:
Total increasing only starts a new cycle when it hits 0 no? Starting a new cycle here would mean that the total count of requests would double

Reply:
I thought it starts a new cycle on a decreased value? From docs „A decreasing value is interpreted as the start of a new meter cycle or the replacement of the meter.“

Reply:
I believe it will also log in the logs that it did something it should not do